transforms/universal/doc_id/ray/Dockerfile

```diff
@@ -10,8 +10,11 @@ RUN cd data-processing-lib-python && pip install --no-cache-dir -e .
 COPY --chown=ray:users data-processing-lib-ray/ data-processing-lib-ray/ 
 RUN cd data-processing-lib-ray    && pip install --no-cache-dir -e .
 
-COPY requirements.txt requirements.txt
-RUN pip install --no-cache-dir -r  requirements.txt
+# Install ray project source
+COPY --chown=ray:users src/ src/
```

Comment:
can it be done in the Makefile by `.defaults.copy-lib` ?

Reply:
Its done there, but we still need to install

Reply:
yes we need to install it, but we don't have to copy each part separately, the temp directory contains only relevant files

Reply:
wait, we are installing local directory. We do not need any makefiles support here

Reply:
Makefile infrastructure is not part of the docker build environment.